client/src/components/categorical/category.js

```diff
@@ -238,58 +238,66 @@ class Category extends React.Component {
               {""}
             </label>
             {/* Dialog uses portal, can be anywhere/factored out */}
```

Comment:
What does this comment mean?

Reply:
See `portalContainer` here: https://blueprintjs.com/docs/#core/components/dialog

It's going to be attached to `document.body`. Similarly, we have componetized our `toasters` and called them in functions. Rendering is independent of the DOM structure, unlike all the other react elements in the component tree.

Reply:
What I'm confused about is that the Dialog was removed by this PR.   So why is the comment still present in the code?